ccd-definition/AuthorisationCaseField/CareSupervision/la-solicitor.json

```diff
@@ -1354,21 +1354,6 @@
     "UserRole": "[LASOLICITOR]",
     "CRUD": "CRU"
   },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseFieldID": "cmoSupportingDocs",
-    "UserRole": "caseworker-publiclaw-solicitor",
-    "CRUD": "CRUD",
-    "Comment": "Requires idam role otherwise cannot add to"
-  },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseFieldID": "cmosSentToJudge",
```

Comment:
why do we want to remove this?

Reply:
Bullseye! That's just a left over, we don't need to remove this. I have reverted this change.